addons/sourcemod/scripting/soap_tf2dm.sp

```diff
@@ -355,8 +356,12 @@ void InitSpawnSys()
         else
         {
             LogMessage("Map spawns missing. Map: %s, no cURL support", map);
-            LogError("File Not Found: %s, no cURL support", path);
-            SetDefaultSpawns(true, true);
+            if (GetConfigPath(map, path, sizeof(path))) {
```

Comment:
Ideally, print something to chat about not finding anything here

Reply:
It now prints to chat if and what fallback cfg is being loaded:

```
[SOAP] No configuration found for cp_steel_f4, loading fallback configuration from cp_steel.cfg.
```

Reply:
Phenomenal, thanks, LGTM

Reply:
Any way you could add a cvar like `soap_use_oldmaps_cfg` or something to control this behavior as well? Other than that should be good for a merge, tysm 👍